### DISCUSSION TITLE
How to set superset so that it can be accessed via subdomain

### DISCUSSION THREAD

**Discussion by EkaX** [apache/superset#28100] [Q&A / Help]
Hello i want to ask how to set superset so that it can be accessed via subdomain
I tried superset on lightsail aws and ubuntu 22.04

I have set a sub domain in config.py and tried to install and setup nginx and the result is I still can't access it via the sub domain

> 
# The base URL to query for accessing the user interface
WEBDRIVER_BASEURL = "http://sub.domain.id:8088/"

Any one can help

**Comment by chuasoonee** [comment]
If you starts your superset using: superset run -p 8088 --reload, by default it's only accessible within localhost.
If you want it to be accessed via subdomain (external), you need to run with: superset run -h 0.0.0.0 -p 8088 --reload. (and configure your firewall to allow port 8088, in my case Debian and using iptables)
Then configure your nginx's proxy_pass to http://127.0.0.1:8088 and server_name to sub.domain.id

Hope this helps.
